### PR TITLE
HiFi voice cloning docs

### PR DESCRIPTION
**Pull request by boli-bohan**

### CODE REVIEW

File: fern/assets/images/hifi-clone.png
Comment (kbrgl) [inline fern/assets/images/hifi-clone.png:1]: "Voice" being capitalized is a bit odd

File: fern/build-with-sonic/capabilities/clone-voices.mdx [hunk 1]
```diff
@@ -14,7 +14,17 @@ For the best voice clones, we recommend following these practices in the source
...
 5. **Avoid long pauses in the clip.** Too many long pauses could result in the cloned voice drifting from the source clip.
 6. **Speak in the target language.** For instance, if you want the cloned voice to speak Spanish, speak Spanish in the recording. If this is not possible, you can use Cartesia's localization feature—available in the playground and in the API—to convert your clone to a different language.
+
+## Hi-Fi Voice Cloning
+
+<Frame background="subtle">
+  <img src="/assets/images/hifi-clone.png" />
+</Frame>
+
+Hi-Fi voice cloning is a flavor of instant cloning for creating higher-fidelity clones. This means higher speaker similarity, but noise and other artifacts from the source clip may also get picked up.
```
Comment (kbrgl) [inline fern/build-with-sonic/capabilities/clone-voices.mdx:28]:
```suggestion
Hi-Fi voice cloning is a type of instant cloning that creates higher fidelity clones, but it's also more sensitive to picking up artifacts and aberrations in the source clip.
```

**Review by kbrgl — APPROVED** [review]

File: fern/assets/images/hifi-clone.png
Comment (boli-bohan) [reply fern/assets/images/hifi-clone.png:1]: Fixed in the UI pr and updated image here

**Comment by bclyang** [comment]
My recommendation is to actually structure this as follows:
* Instant cloning - there's two sections: High Stability and High Similarity
* For High Stability - Results in a highly stable and clear voice, even if the input is noisy. Can use up to 30 seconds of audio. No transcripts required.
* For High Similarity - Results in a voice that closely resembles the input in style, noise, and pacing. Can only use up to 10 seconds of audio, ideally just a single sentence. Transcripts optional - if possible to provide exactly correct transcripts, would be helpful. Artifacts in the audio (e.g. cutting off early, etc.) can result in artifacts in the resulting generation.

**Pull request closed without merging**